### ISSUE TITLE
[1.12.2] Recipe not showing up properly in JEI

### ISSUE THREAD

**Issue by SeriousCreeper** [codetaylor/artisan-worktables#12]
Artisan version: 1.5.8
JEI version: tested 4.8.5.140 and 4.8.5.143
Forge version: 2584

Heya,

i added a recipe for the vanilla crafting table to the carpenters bench. When i look it up in JEI it doesn't display it properly (looks like it chooses a shapeless version) but when i try to actually craft it, it works properly (using the shape i see in JEI won't craft anything).

Script i used:
```
Worktable.addRecipeShaped("carpenter", <minecraft:crafting_table>, <ore:carpenters_handsaw>, 10, true, [
	    [<ore:plankWood>, <ore:plankWood>],
	    [<ore:plankWood>, <ore:plankWood>]]);
```

JEI view: 
![](https://i.imgur.com/b71LTuP.png)

When i actually try to craft it:
![](https://i.imgur.com/K1myXB5.png)

Let me know if there is anything else i can provide to help! 
Really loving the mod btw :D

**Comment by codetaylor** [comment]
Confirmed.

![image](https://user-images.githubusercontent.com/1240716/34646446-4c44848c-f31d-11e7-8e47-d0697ce94cd9.png)

**Comment by codetaylor** [comment]
Confirmed fixed.

![image](https://user-images.githubusercontent.com/1240716/34646707-2c5e69ba-f324-11e7-8a12-0dc1d5bfd171.png)

**Comment by codetaylor** [comment]
Fixed in version 1.12.2-1.5.9.

Thanks for the awesome bug report! Seriously. :)
